public/index.html

```diff
@@ -4,11 +4,10 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <script src="https://www.lactame.com/lib/ml/4.0.0/ml.min.js"></script>
```

Comment:
I think I need this script to make the KNN algorithm work. Can we add it back or did it cause problems?

Reply:
oh, I see, you moved it into `KNNTrainer` 👍

Reply:
Yes we now use a published version.